backend/requirements.txt

```diff
@@ -12,5 +12,7 @@ redis==3.4.0
 ruamel.yaml==0.16.5
 six==1.13.0
 slacker-log-handler==1.7.1
+socketIO-client==0.7.2
```

Comment:
why are these dependencies needed for the backend?

Reply:
because exabgp-based mitigation (which should be available as I see it with the user requests), requires socketio for communication with the container. We run mitigation in backend, hence the backend should be able to support this with the proper packages.